.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,6 +1,6 @@
 name: Feature Request
 description: Request a new feature that you would like to be part of the Postman platform. 
-title: "<title>"
+title: "[Feature] "
```

Comment:
I find this is helpful to more easily visually scan alongside the label.

Reply:
That was just there as a placeholder which users should replace with the title of their issue/feature request.